index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <!--<section id="portfolio" class="project-section">-->
+            <div>
```

Comment:
- [ ] The container of the works section uses a section tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. Inside it will be all the project cards where each can be wrapped in a `<div>` or better still an `<article>` tag. 
- [ ] The main container (the section tag I mentioned above) of the project cards has a `display:grid` rule used to align it content. 

**_Kindly fix this_**